test/functional/Java19andUp/build.xml

```diff
@@ -82,7 +82,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<target name="build">
 		<if>
 			<not>
-				<matches string="${JDK_VERSION}" pattern="^(8|9|10|11|12|13|14|15|16|17|18)$$" />
+				<matches string="${JDK_VERSION}" pattern="^(8|9|10|11|12|13|14|15|16|17|18|19)$$" />
```

Comment:
This is just wrong: the folder applies to Java 19 and up, but you could simplify the pattern:
```
				<matches string="${JDK_VERSION}" pattern="^(8|9|1[0-8])$$" />
```

Reply:
Fixed as suggested above.